core/src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/EntityComponentSpec.scala

```diff
@@ -15,6 +15,7 @@ import java.util.UUID
  */
 class EntityComponentSpec extends TestDriverComponentWithFlatSpecAndMatchers with RawlsTestUtils {
   import driver.api._
+  implicit object SetUUIDParameter extends SetParameter[UUID] { def apply(v: UUID, pp: PositionedParameters) { pp.setBytes(uuidColumnType.toBytes(v)) } }
```

Comment:
instead of duplicating this via copy/paste, how about adding ` with RawSqlQuery` to the `EntityComponentSpec` definition at line 16? (this would allow you to remove the extra imports at line 9 too)